sweddit_combined/history/countries/CHC - Chu.txt

```diff
@@ -1,7 +1,41 @@
 government = monarchy
-add_government_reform = chinese_warlord
+add_government_reform = chinese_hegemon_reform
 technology_group = chinese
 religion = confucianism
-primary_culture = xiang
+primary_culture = hubei
+add_accepted_culture = sichuanese
 capital = 682	#Wuchang
-
+1421.04.28 = { #Coronation
+	monarch = {
+		name = Zhuang
+		dynasty = Chu
+		adm = 4
+		dip = 3
+		mil = 5
+		birth_date = 1396.11.28
+		female = no
+		regent = no
+		religion = confucianism
+		culture = hubei
+	}
+	clear_scripted_personalities = yes
+	add_ruler_personality = inspiring_leader_personality
+	add_ruler_personality = drunkard_personality
+	add_ruler_personality = architectural_visionary_personality
+}
```

Comment:
Should these really be set? Not a big thing i guess but still wasn't specified.

Reply:
Annima says ruler traits were in the proposal